xray/services/xraywatch.go

```diff
@@ -0,0 +1,186 @@
+package services
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	rthttpclient "github.com/jfrog/jfrog-client-go/artifactory/httpclient"
+	// "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+
+	artUtils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/auth"
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/jfrog/jfrog-client-go/xray/services/utils"
+)
+
+const (
+	watchAPIURL = "api/v2/watches"
+)
+
+// XrayWatchService defines the http client and xray details
+type XrayWatchService struct {
```

Comment:
Ket's rename the struct to ```WatchService``` and this file to ```watch.go```.
Let's also rename ```NewXrayWatchService``` to ```NewWatchService```.

Reply:
@eyalbe4 

Most, if not all, structs have a prefix of `Xray`.
Did you want them all to be renamed without Xray?

Reply:
Hi @eyalbe4, I've renamed all structs & functions that included Xray.